src/client/cli/cmd/set.cpp

```diff
@@ -63,22 +63,33 @@ QString cmd::Set::description() const
 
 mp::ParseCode cmd::Set::parse_args(mp::ArgParser* parser)
 {
-    parser->addPositionalArgument("key", "Path to the option to configure.", "<key>");
-    parser->addPositionalArgument("value", "Value to set the option to.", "<value>");
+    parser->addPositionalArgument(
+        "keyval",
+        "A key-value pair. The key specifies a path to the option to configure. The value is its intended value.",
+        "<key>=<value>");
 
     auto status = parser->commandParse(this);
     if (status == ParseCode::Ok)
     {
         const auto args = parser->positionalArguments();
-        if (args.count() == 2)
+        if (args.isEmpty())
```

Comment:
If someone provides more than one, it will be silently dropped?

Reply:
Good catch, I did not do this because I meant to evolve to multiple arguments and then forgot.